### DISCUSSION TITLE
Elastic agent not returning osquery results

### DISCUSSION THREAD

**Discussion by ihuxn4hi** [Security-Onion-Solutions/securityonion#10217] [2.4]
I'm doing a 'SELECT * FROM users;' from the canned osquery queries and it's just spinning with no results returned from any of the hosts (two SO hosts plus one Amazon Linux host).

<img width="1430" alt="image" src="https://user-images.githubusercontent.com/123765379/234443961-76a6cc19-53d6-4ac2-8836-26faf01a5e23.png">

Other than open 8220 in Administration > Configuration > firewall > hostgroups > Elastic agents, is there anything you need to do to get this to work?

**Comment by dougburks** [comment]
Have you tried Security Onion 2.4.2 released last week?
https://blog.securityonion.net/2023/06/security-onion-24-beta-3-release-now.html

![Screenshot 2023-06-05 at 10 34 49 AM](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/f50197b3-16cc-4e4b-932e-80593b26ed23)
